en/docs/learn/extension-points-for-oauth.md

```diff
@@ -8,12 +8,15 @@ points related to OAuth and OpenID Connect (OIDC).
 Any OAuth extension listed below can be implemented, by implementing an
 interface or extending an abstract class as defined under each extension
 point.  
-When the implementation is done, package your classes as a .jar file and
-place it in the *\<IS\_HOME\>/repository/component/lib* directory.  
-Then configure your extension in
-*<IS\_HOME\>/repository/conf/deployment.toml* file referring to the ' *Configuration* ' section of the
-respective extension point below.  
-Restart the server to effect changes.
+
+1. When the implementation is done, package your classes as a .jar file and
+place it in the `<IS_HOME>/repository/component/lib` directory.  
+
+2. Next, configure your extension in
```

Comment:
extension in the

Reply:
Fixed in 6cad0bf12e3eca4ca70d9dea7b04128ea0b02680